README.md

```diff
@@ -1 +1,106 @@
+  - Use JavaScript to manipulate DOM elements.
+  - Use JavaScript events.
+  - Use objects to store and access data.
+
+- GitHub
+  - Use GitHub Pages to deploy web pages.
+
+## Live Demo
+
+- [Live link](https://https://187jjay187.github.io/cap-tech-page/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Browser and git
+
+### Install
+git clone https://github.com/Shaheryar0054/First-Capstone-Project
+
+### Usage
+Open in browser
+
+### Project Documantaion and short descriptive Video
+
+- Click on the link for [Short video](https://www.loom.com/share/3a9a3e10d10d429b89f2db89be4cb199)
+## Authors
+
+👤 **Jerome**
+
+- GitHub: [@githubhandle](https://github.com/187jjay187)
+- Twitter: [@twitterhandle](https://twitter.com/187jjay187)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jerome-osman-137605a4)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/187jjay187/cap-tech-page/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to the Microverse team for the great curriculum.
+- Thanks to the Code Reviewer(s) for the insightful feedback.
+- A great thanks to My coding partner(s), morning session team, and standup team for their contributions.
+- Hat tip to anyone whose code was used.
+- Cindy Shin the author of the original design [www.behance.net](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015/)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
+Copyright (c) 2022 Mr. J. Osman.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```

Comment:
- [ ] I think it will be better to create a new MD file for your licence and link it to your readme instead put it in your readme in order to make it more professional :+1: